x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/policies/package_policies.tsx

```diff
@@ -98,35 +103,46 @@ export const PackagePoliciesPage = ({ name, version }: PackagePoliciesPanelProps
     [setPagination]
   );
 
-  const renderViewDataStepContent = useCallback(
-    () => (
-      <>
-        <EuiText>
-          <FormattedMessage
-            id="xpack.fleet.agentEnrollment.viewDataDescription"
-            defaultMessage="After your agent starts, you can view your data in Kibana by using the integration's installed assets. {pleaseNote}: it may take a few minutes for the initial data to arrive."
-            values={{
-              pleaseNote: (
-                <strong>
-                  {i18n.translate(
-                    'xpack.fleet.epm.agentEnrollment.viewDataDescription.pleaseNoteLabel',
-                    { defaultMessage: 'Please note' }
-                  )}
-                </strong>
-              ),
-            }}
-          />
-        </EuiText>
-        <EuiSpacer size="l" />
-        <EuiButton href={getHref('integration_details_assets', { pkgkey: `${name}-${version}` })}>
-          {i18n.translate('xpack.fleet.epm.agentEnrollment.viewDataAssetsLabel', {
-            defaultMessage: 'View assets',
-          })}
-        </EuiButton>
-      </>
-    ),
-    [name, version, getHref]
-  );
+  const viewDataStep = useMemo<EuiStepInterface>(() => {
+    if (agentEnrollmentFlyoutExtension) {
+      return {
+        title: agentEnrollmentFlyoutExtension.title,
+        children: <agentEnrollmentFlyoutExtension.Component />,
```

Comment:
@nchaulet - Here's the "other end" of the UI Extension for the flyout. Much less of a "special case" because we just choose where to render `Component`.